lnbits/core/crud.py

```diff
@@ -404,6 +404,47 @@ async def get_payments(
     )
 
 
+async def get_payments(
+    *,
+    wallet_id: Optional[str] = None,
+    complete: bool = False,
+    pending: bool = False,
+    outgoing: bool = False,
+    incoming: bool = False,
+    since: Optional[int] = None,
+    exclude_uncheckable: bool = False,
+    filters: Optional[Filters[Payment]] = None,
+    conn: Optional[Connection] = None,
+    limit: Optional[int] = None,
+    offset: Optional[int] = None,
+) -> list[Payment]:
+    """
+    Filters payments to be returned by complete | pending | outgoing | incoming.
+    """
+
+    if not filters:
+        filters = Filters()
+
+    if limit:
+        filters.limit = limit
+    if offset:
+        filters.offset = offset
+
+    page = await get_payments_paginated(
```

Comment:
do we want to  return paginated here aswell?

Reply:
I didn't want to duplicate all the code, the functionality is the same